error_tracker/django/models.py

```diff
@@ -34,8 +34,18 @@ class ErrorModel(models.Model, ModelMixin):
     ticket_raised = models.BooleanField(default=False)
 
     @classmethod
-    def get_exceptions_per_page(cls, page_number=1):
-        records = cls.objects.all().order_by('last_seen')
+    def get_exceptions_per_page(cls, query):
```

Comment:
**query

Reply:
Hello,

This is not a kwargs but a true dict i will reuse as filter:
```
if not query:
            records = cls.objects.all().order_by('last_seen')
 else:
            records = cls.objects.filter(**query).order_by('last_seen')
```

I will submit a commit with a fix on the jquery in the template. Url parameter are not well formatted when multiple parameter, so this dict, is not well formatted too.

Reply:
This method is different from super class(`ModelMixin`), if someone has implemented via keyword argument page_number. It'll start throwing errors. 

That's the reason I'm suggesting this method to take kwargs as parameter. We should not break backward compatibility.

Reply:
Ok let me see how i could manage this...